src/plugin/plugin.rs

```diff
@@ -223,11 +225,23 @@ where
                     PhysicsSet::Writeback,
                 )
                     .chain()
-                    .before(TransformSystem::TransformPropagate),
+                    .before(TransformSystem::TransformPropagate)
+                    .after(systems::sync_removals),
```

Comment:
interesting change, do you remember the rationale ?

Reply:
If I remember correctly, I did that to prevent any of the init_* systems from running before sync_removals had finished. I believe I was running into inconsistencies in that regard without this being set, causing stuff to be removed after it had been re-added to the proper world.